### PR TITLE
Object sticking

### PR DESCRIPTION
**Pull request by sophiahxu**

### CODE REVIEW

File: core/src/edu/cornell/gdiac/bubblegumbandit/controllers/CollisionController.java [hunk 1]
```diff
@@ -245,35 +256,159 @@ public void resolveGumCollision(Obstacle bodyA, Obstacle bodyB){
...
         if (gum != null && gum.getName().equals("gumProjectile")) {
             // Do this once gum is turning from a projectile to sticky
             gum.setVX(0);
             gum.setVY(0);
             gum.setTexture(bubblegumController.getStuckGumTexture());
             gum.setName("stickyGum");
-            gum.setRadius(gum.getRadius() * 1.5f);
+            gum.setRadius(gum.getRadius());
             // Changing radius resets filter for some reason
             gum.getFilterData().maskBits = MASK_GUM;
```
Comment (agettig) [inline core/src/edu/cornell/gdiac/bubblegumbandit/controllers/CollisionController.java:283]: Do we still need to reset filter data if we are no longer changing the radius

File: core/src/edu/cornell/gdiac/bubblegumbandit/controllers/CollisionController.java [hunk 2]
```diff
@@ -245,35 +256,159 @@ public void resolveGumCollision(Obstacle bodyA, Obstacle bodyB){
...
+                if (!gum.onTile()) {
+                    gum.markRemoved(true);
+                    enemy.setGummedTexture();
+                    enemy.setGummed(true);
+                    gummedRobots.addLast(enemy);
+                }
+                else {
+                    enemy.setStuck(true);
+                }
+            }
```
Comment (agettig) [inline core/src/edu/cornell/gdiac/bubblegumbandit/controllers/CollisionController.java:299]: What's gummed vs stuck?
Are they mutually exclusive?

**Review by agettig — COMMENTED** [review]
Add Javadoc comments

File: core/src/edu/cornell/gdiac/bubblegumbandit/controllers/CollisionController.java (hunk 2 above)
Comment (sophiahxu) [reply core/src/edu/cornell/gdiac/bubblegumbandit/controllers/CollisionController.java:299]: gummed is when the robot is hit w gum and turned pink. Stuck is when the robot just hits gum that's on a tile and gets stuck. I differentiated them bc I don't want the gum disappearing if it's just stuck and not gummed.

File: core/src/edu/cornell/gdiac/bubblegumbandit/controllers/CollisionController.java (hunk 1 above)
Comment (sophiahxu) [reply core/src/edu/cornell/gdiac/bubblegumbandit/controllers/CollisionController.java:283]: mm yea I can clean this up

Comment (sophiahxu) [reply core/src/edu/cornell/gdiac/bubblegumbandit/controllers/CollisionController.java:283]: pretty sure these are still required after messing w it

**Review by agettig — APPROVED** [review]

**Pull request closed without merging**